controllers/articles.controller.js

```diff
@@ -31,3 +32,14 @@ exports.getCommentsByArticleId = (req, res, next) => {
 		})
 		.catch(next)
 }
+exports.patchArticle = (req, res, next) => {
+	const { article_id } = req.params
+	const { inc_votes } = req.body
+	if (article_id && typeof inc_votes === 'number') {
```

Comment:
You should have some error handling functions that would take care of psql errors such as incrementing by the wrong datatype